mainnet-canisters.json

```diff
@@ -20,24 +20,24 @@
     "sha256": "e6072806ae22868ee09c07923d093b1b0b687dba540d22cfc1e1a5392bfcca46"
   },
   "cycles-minting": {
-    "rev": "77f48ae63af09b6538b1bf33d3accc3bc74d14f8",
-    "sha256": "3260e795bd3e446a189539ce89d44cb29f7d196b92cdd2e2c75571c062ef1e50"
+    "rev": "ca8847547d327ce8a3bd81d25a590e01da1a3af5",
+    "sha256": "57f2cbbdbc2d8943173acc6ebd55aca9b9c12e2140b38ff5f20aea7b101ea570"
   },
```

Comment:
PR description does not mention cycles-minting. I guess we forgot to update this after previous release? We really need a robot to generate these PRs automatically...

Reply:
Good catch. I've added a link to the CMC proposal